charts/substra-backend/values.yaml

```diff
@@ -197,3 +197,13 @@ celeryworker:
   tolerations: []
 
   affinity: {}
+
+extraEnv:
+  - name: CHAINKEYS_FOR_TRAINING_TASKS_ENABLED
+    value: False
+  - name: NODE_INDEX
+    value: 1
+  - name: K8S_SECRET_NAMESPACE
+    valueFrom:
+      fieldRef:
+        fieldPath: metadata.namespace
```

Comment:
the default values should be empty in the opensourced chart. Put those specific values in the infrastructure repo